test/lib/CMakeLists.txt

```diff
@@ -15,6 +15,12 @@ if (UFE_FOUND AND MAYA_APP_VERSION VERSION_GREATER 2020)
     )
 endif()
 
+if (UFE_FOUND AND MAYA_APP_VERSION VERSION_GREATER_EQUAL 2022)
+    list(APPEND TEST_SCRIPT_FILES
+        testMayaUsdDirtyScene.py
+    )
+endif()
```

Comment:
The UFE scene dirty was only added in Maya 2022, so this test can only be done there.